totalcommander.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": "9.20",
+    "license": "Shareware",
+    "homepage": "https://www.ghisler.com",
+    "description": "Total Commander is a Shareware file manager for Windows® 95/98/ME/NT/2000/XP/Vista/7/8/8.1/10, and Windows® 3.1.",
+    "architecture": {
+        "64bit": {
+            "url": "http://totalcommander.ch/win/tcmd920x64.exe",
+            "hash": "1887c4cb3498572965ad813c6b8c7888514f515b9acba14a6618111db1a43260",
+            "bin": "TOTALCMD64.EXE",
+            "shortcuts": [
+                [
+                    "TOTALCMD64.EXE",
+                    "Total Commander"
+                ]
+            ]
+        },
+        "32bit": {
+            "url": "http://totalcommander.ch/win/tcmd920x32.exe",
+            "hash": "90b7f759df3a7e35c0511b1402508d265704b120622dcd4effb6aa74c238f2c6",
+            "bin": "TOTALCMD.EXE",
+            "shortcuts": [
+                [
+                    "TOTALCMD.EXE",
+                    "Total Commander"
+                ]
+            ]
+        }
+    },
+    "pre_install": [
+        "extract_7zip \"$dir\\$fname\" \"$dir\\_tmp\" $false",
+        "extract_7zip \"$dir\\_tmp\\INSTALL.CAB\" \"$dir\" $true",
+        "Move-Item \"$dir\\e\\*\" \"$dir\"",
+        "Remove-Item -Recurse \"$dir\\d\", \"$dir\\e\", \"$dir\\_tmp\", \"$dir\\$fname\"",
```

Comment:
_tmp should exists after robocopy, fname could be removed when you use something like

Reply:
That's what I used before I created this PR. Reason why I chose to extract installer to its own directory `_tmp` was because it's easier to clean up afterward. If you let scoop unpack the installer to default directory by rewriting filename, it will unpack other files along with `INSTALL.CAB` as well, which you would have to selectively remove afterward.